tests/unit/model/test_pose.py

```diff
@@ -0,0 +1,26 @@
+from datachain.model.pose import Pose, Pose3D
+
+
+def test_pose():
+    pose = Pose(x=[10, 20, 30], y=[40, 50, 60])
+    assert pose.model_dump() == {
+        "x": [10.0, 20.0, 30.0],
```

Comment:
should these be ints?

Reply:
Since `Pose` is now a generic bounding box model I believe we should have `floats` here. See also my comment [here](https://github.com/iterative/datachain/pull/924/files/fdc7f1b033dd6845caf380a8884949a5509cf372#r1957254868) and model declaration [here](https://github.com/iterative/datachain/pull/924/files/fdc7f1b033dd6845caf380a8884949a5509cf372#diff-86234d5135b709fdb026741ca44df7a33d0dbdc127593b1d50b87a94c3c59217R10-R23).